docusaurus/docs/tutorial/01.md

```diff
@@ -3,213 +3,442 @@ id: 01
 title: 01. Creating Your First Page
 sidebar_label: 01. Creating Your First Page
 ---
-In this section, we will try to create a new page on our Treats project.  
-There are two main steps in this tutorial:  
-1. Creating your page and style it
-2. Register your page to application route
+__Treats using React as it's core. If you don't get a grip on React concept, we suggest you to learn more about [React][React] before following this tutorial.__
 
+In this section, we will try to create a simple `todo-apps` using Treats.
+Install __`create-treats-app`__ packages and run the command.
```

Comment:
why I need to install and run that command?

suggestion : add the goals, for this case "to create/bootstrap treats app`